### ISSUE TITLE
Uncaught error when selected language for the word does not exist

### ISSUE THREAD

**Issue by C0rn3j** [Suyash458/WiktionaryParser#31]
The word 'behagelig' for example has only a definition in Norwegian Bokmål.

https://en.wiktionary.org/wiki/behagelig#Norwegian_Bokm%C3%A5l

If I try to query for the language 'Norwegian Nynorsk' for example:
```
parser.fetch("behagelig", "norwegian nynorsk")
```
I get a following error:
```
Traceback (most recent call last):
  File "./scrap.py", line 24, in <module>
    another_word = parser.fetch(wordToScrap, NN)
  File "/home/c0rn3j/wikivirtpython/lib/python3.6/site-packages/wiktionaryparser/WikiParse.py", line 233, in fetch
    return self.get_word_data(language.lower())
  File "/home/c0rn3j/wikivirtpython/lib/python3.6/site-packages/wiktionaryparser/WikiParse.py", line 84, in get_word_data
    if index.startswith(start_index):
TypeError: startswith first arg must be str or a tuple of str, not NoneType
```

**Comment by Suyash458** [comment]
Seems fixed in 39ba27422ab33e104d0f034df1e62848a5229c48

**Comment by C0rn3j** [comment]
Looks like it, thanks @duffrecords!
